scripts/install-frontend.sh

```diff
@@ -0,0 +1,4 @@
+cd src-frontend
+npm install -g bower
+npm install
+bower install
```

Comment:
That's actually why I made this a separate script for `bash -x`. It won't affect the environment of the caller.